### PR TITLE
#153756398 Integrate HoundCI 

### PR DESCRIPTION
**Pull request by hay-bams**
#### What does this PR do?
Integrate HoundCI with badge attached to readme

#### Description of Task to be completed?
Integrate HoundCI to automatically validate the codebase with standard style guides

#### How should this be manually tested?
Badge is visible in README file

#### Any background context you want to provide?
N/A

#### Screenshots
screen shot 2017-12-21 at 2 14 42 pm

#### Questions:
N/A

### CODE REVIEW

File: .hound.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+eslint:
+  enabled: true
+  config_file: .eslintrc
```
Comment (temilaj) [inline .hound.yml]: We currently do not have this ESLint config in our application. Where are you getting this from?

Comment (hay-bams) [reply .hound.yml]: It's a snippet I got from the houndCI documentation to configure hound to use ESlint as the s tyle guide for our application when ESlint gets configured for our application

Comment (temilaj) [reply .hound.yml]: Yes, but it says to point to a configuration file. When you read the documentation, you should strive towards understanding exactly how the product works, and not just use the snippet.

Comment (hay-bams) [reply .hound.yml]: okay, got it.

**Pull request closed without merging**